pkg/chunk/cached_store.go

```diff
@@ -604,6 +611,7 @@ type Config struct {
 	Compress       string
 	MaxUpload      int
 	Writeback      bool
+	DelayUpload    string
```

Comment:
It's better to use number seconds here, move the parse into cmd/

Reply:
`time.Duration` is better?  I will change to time.Duration